rules_python/piptool.py

```diff
@@ -192,6 +192,26 @@ def extract(args):
           contents.append(f.read() + '\n')
   contents = '\n'.join(contents)
 
+  parser = whl.entrypoints()
+  entrypoints_build = ''
+  if parser:
+      if parser.has_section('console_scripts'):
+          for name, location in parser.items('console_scripts'):
+              # Assumes it doesn't depend on extras. TODO(conrado): fix
+              entrypoint_file = 'entrypoint_%s.py' % name
+              with open(os.path.join(args.directory, entrypoint_file), 'w') as f:
+                  f.write("""from %s import %s as main; main()""" % tuple(location.split(":")))
+              entrypoints_build += """
+py_binary(
+    name = "{entrypoint_name}",
```

Comment:
It feels that the "entrypoint_tensorboard" naming convention should be abstracted away from the user.
How about this:
```
--- a/rules_python/piptool.py
+++ b/rules_python/piptool.py
@@ -542,7 +542,9 @@ requirements_map = _requirements
 def requirement_repo(name):
   return requirement(name).split(":")[0]
 
-def requirement(name):
+def requirement(name, binary=None):
+  if binary:
+    return requirement_repo(name) + ":entrypoint_" + binary
   key = name.lower()
   if key not in _requirements:
     fail("Could not find pip-provided dependency: '%s'" % name)
```
Then the users could do
```
requirement("tensorboard", binary="tensorboard")
```
to get to the py_binary target?